editions/tw5.com/tiddlers/concepts/DefaultTiddlers.tid

```diff
@@ -1,10 +1,15 @@
 created: 20180306113011255
-modified: 20180306161225140
+modified: 20241104150049419
 tags: Concepts
 title: DefaultTiddlers
 type: text/vnd.tiddlywiki
 
-"Default tiddlers" refers to the list of tiddlers that is opened at startup in the browser if the address bar does not include a [[permalink|PermaLinks]]).
+"Default tiddlers" refers to the list of tiddlers that is opened at startup in the browser if the address bar does not include a [[permalink|PermaLinks]].
+
+There are 2 ways default tiddlers can be defined:
```

Comment:
Normal practice would be only use numerals for larger numbers, and spell out "two" here.